es6.md

```diff
@@ -189,3 +189,48 @@ interface ImportNamespaceSpecifier {
 ```
 
 A namespace import specifier, e.g., `* as foo` in `import * as foo from "mod.js"`.
+
+## ExportNamedDeclaration
+
+```js
+interface ExportNamedDeclaration <: Node {
+    declaration: Declaration | null;
+    specifiers: [ ExportSpecifier ];
+    source: Literal | null;
+}
+```
+
+An export named declaration, e.g., `export {foo, bar};`, `export {foo} from "mod";` or `export var foo = 1;`.
+
+_Note: Having `declaration` populated with non-empty `specifiers` or non-null `source` results in an invalid state._
+
+## ExportSpecifier
+
+```js
+interface ExportSpecifier {
+    exported: Identifier;
+    local: Identifier;
+}
+```
+
+An exported variable binding, e.g., `{foo}` in `export {foo}` or `{bar as foo}` in `export {bar as foo}`. The `exported` field refers to the name exported in the module. The `local` field refers to the binding into the local module scope. If it is a basic named export, such as in `export {foo}`, both `exported` and `local` are equivalent `Identifier` nodes; in this case an `Identifier` node representing `foo`. If it is an aliased export, such as in `export {bar as foo}`, the `exported` field is an `Identifier` node representing `foo`, and the `local` field is an `Identifier` node representing `bar`.
+
+## ExportDefaultDeclaration
+
+```js
+interface ExportDefaultDeclaration <: Node {
+    declaration: Declaration | Expression;
+}
+```
+
+An export default declaration, e.g., `export default function () {};` or `export default 1;`.
+
+## ExportBatchDeclaration
```

Comment:
Just spoke to @ariya and a few others and they suggested `ExportAllDeclaration` is less literal and perhaps more clear than `ExportStartDeclaration` -- which is fine with me. So allow me to update my bikeshed to suggest that instead.

Looking for +1s/-1s from everyone so we can move forward with the change, or leave as-is.